react/features/base/tracks/actions.js

```diff
@@ -79,7 +81,11 @@ export function createLocalTracksA(options = {}) {
         // to implement them) and the right thing to do is to ask for each
         // device separately.
         for (const device of devices) {
-            createLocalTracksF(
+            if (getState()['features/base/tracks']
```

Comment:
this change can go separate from the fix, right?

Reply:
no it's part of the fix, why do you think it would go separate ? it's just an assertion

Reply:
Just asking :-) It may make the review faster since the change would be smaller.